lib/models/application.ts

```diff
@@ -295,6 +344,7 @@ const getApplicationModel = function (
 					options: mergePineOptions(
 						{
 							$filter: {
+								...accessFilter,
```

Comment:
```suggestion
								...accessFilter.$filter,
```
... if the suggestions in the previous comments were adopted (`$filter` may be undefined, but `...undefined` works fine).